constants.py

```diff
@@ -106,8 +106,9 @@ class ServoExtension(Enum):
 
 # Free fall to Landing:
 
-# Consider the rocket to have landed if it is within 15 meters of the launch site height.
-GROUND_ALTITUDE = 15.0  # meters
+# Consider the rocket to have landed if it is within 15 meters of the launch site height and the speed is low.
+GROUND_ALTITUDE = 10.0  # meters
+LANDED_SPEED = 5.0  # m/s
```

Comment:
I almost did this as well, but then our vertical velocity still showed -15m/s ish after landing (interest launch), so I was not sure if that's a good measure. Maybe if we give it enough time the velocity would come down, it's just that the data ended prematurely so I can't be sure.

Reply:
It showed this because we cut off the launch data before it actually landed I think. I think we should keep something like this and just guess the values, because worst case it doesn't do the state change and we just get all the data to log.

Reply:
For the interest launch we definitely got touchdown (spike in accel). For purple nurple yeah idk it might have been cutoff

Reply:
Maybe then for interest launch my file I was using for the sim got cut off at the end? You can see the slope of the altitude is not zero at the end.

Reply:
I think you're using the wrong data? The graph in the readme shows LandedState.

Reply:
oh it might have been will's new data also, I haven't looked at that

Reply:
No my data is the same. the graph in the readme incorrectly detects the landed state, you can see it is still sloping down. Landed state should have a slope of 0.

Reply:
> incorrectly detects

yeah it intentionally switches over at 15m, so the slope won't be zero. But we can confirm that we actually got touchdown because there is a spike in the acceleration values for both the launches in the landed state. (Run the sim from my branch with `-f` and `-l` and scroll through LandedState accel values)

Reply:
But, yeah I get your point that we shouldn't switch over to LandedState arbitrarily like this. imo acceleration values should be the easiest (and safest?) way to detect landing, irrespective of height of the site. But maybe we can test with the IMU first to see what the rotated velocity values are like 1 second after landing